procs/score/asp08.js

```diff
@@ -21,7 +21,6 @@ exports.scorer = acts => {
     focAll: '',
     focInd: '',
     focOp: '',
```

Comment:
This doesn't block, but I've remembered something else.

We should avoid changing the existing scoring procedures, because they'll change in the upstream repo and potentially cause conflict. We should probably duplicate this one, then modify the scoring used in our script.